### DISCUSSION TITLE
On cache hit create symbolic links when using read_only option

### DISCUSSION THREAD

**Discussion by xyzmurali** [ccache/ccache#1495] [Ideas]
Hi, When using cache as read_ony, is there a possibility to create soft links to the cached objects instead of copying the objects to the output directory?. This can help reduce the size of the  output directory where space is a constraint.

**Comment by jrosdahl** [comment]
Ccache does currently not support creating symbolic links, but it would be possible to implement such support.

However, as you might already know, the downside of using symbolic links is that they will stop working if the referred object files are evicted from the cache. If the cache and the build directory are on the same filesystem, using hard links is better and already supported.
